lib/generators/doorkeeper/templates/migration.rb

```diff
@@ -12,8 +12,9 @@ def change
     add_index :oauth_applications, :uid, unique: true
 
     create_table :oauth_access_grants do |t|
-      t.integer  :resource_owner_id, null: false
-      t.integer  :application_id,    null: false
+      t.integer    :resource_owner_id, null: false
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.